sql/database.sql

```diff
@@ -8294,3 +8296,16 @@ CREATE TABLE `immunization_observation` (
   PRIMARY KEY (`imo_id`)
 ) ENGINE=InnoDB;
 -- --------------------------------------------------------
+-- --------------------------------------------------------
+--
+-- Table structure for table 'calendar external'
+--
+CREATE TABLE calendar_external (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `date` DATE NOT NULL,
```

Comment:
If date is set to NOT NULL, then what data is placed there when it isn't set?(or is this even an issue? meaning is a date always set?)